CMakeLists.txt

```diff
@@ -43,73 +50,76 @@ option(ENABLE_DISCORD_RPC "Enables the Discord Rich Presence feature" ON)
 
 # input backends
 if (WIN32)
-  option(ENABLE_XINPUT "Enables the usage of XInput" ON)
-  option(ENABLE_DIRECTINPUT "Enables the usage of DirectInput" ON)
-  add_definitions(-DHAS_DIRECTINPUT)
+	option(ENABLE_XINPUT "Enables the usage of XInput" ON)
+	option(ENABLE_DIRECTINPUT "Enables the usage of DirectInput" ON)
+	add_compile_definitions("HAS_DIRECTINPUT")
 endif()
 option(ENABLE_SDL "Enables the SDLController backend" ON)
 
 # audio backends
 if (WIN32)
-  option(ENABLE_DIRECTAUDIO "Enables the directaudio backend" ON)
-  option(ENABLE_XAUDIO "Enables the xaudio backend" ON)
+	option(ENABLE_DIRECTAUDIO "Enables the directaudio backend" ON)
+	option(ENABLE_XAUDIO "Enables the xaudio backend" ON)
 endif()
 option(ENABLE_CUBEB "Enabled cubeb backend" ON)
 
 option(ENABLE_WXWIDGETS "Build with wxWidgets UI (Currently required)" ON)
 
-find_package(SDL2 CONFIG REQUIRED)
-find_package(CURL CONFIG REQUIRED)
-find_package(pugixml CONFIG REQUIRED)
-find_package(imgui CONFIG REQUIRED)
-find_package(RapidJSON CONFIG REQUIRED)
+set(THREADS_PREFER_PTHREAD_FLAG true)
+find_package(Threads REQUIRED)
+find_package(SDL2 REQUIRED)
+find_package(CURL REQUIRED)
+find_package(pugixml REQUIRED)
+find_package(imgui REQUIRED)
+find_package(RapidJSON REQUIRED)
 find_package(Boost COMPONENTS program_options filesystem nowide REQUIRED)
 find_package(libzip REQUIRED)
 find_package(glslang REQUIRED)
 find_package(ZLIB REQUIRED)
-find_package(zstd CONFIG REQUIRED)
+find_package(zstd REQUIRED)
```

Comment:
Please add a fallback with pkg-config for zstd. Not all systems have the needed file for CMake.